# A die is thrown until the first time the total sum of the face value of the die is 700 or greater

# Estimate the probability that, for this to happen,

## a) more than 210 tosses are required

## b) less than 190 tosses are required

## c) Between 190 and 210 tosses (inclusive) are required

_______

# $S_{n} = X_{1} + X_{2} + ... + X_{n}$

# $E(S_{n}) = n\frac{7}{2}$

# $V(S_{n}) = nV(X_{i}) = n\frac{35}{12} \implies \sigma_{S_{n}} = \sqrt{n\frac{35}{12}}$

______

# a)

# Let's not over complicate the problem

# What does it mean if it takes more than 210 rolls? 

# It means that $S_{210}<700$, which isn't hard to calculate

# $E(S_{210}) = 210 \cdot \frac{7}{2} = 735$

# $\sigma_{S_{210}} = \sqrt{210 \cdot \frac{35}{12}} = 24.75$

# $\frac{700-735}{24.75} = -1.414141$

# $\implies P(S_{210}<700) = \int_{-\infty}^{-1.4141}\phi(x)dx$

In [18]:
from scipy.stats import norm

In [22]:
norm.cdf(-1.414141)

0.07866025351868966

# Therefore the probability of it requiring more than 210 rolls is about 0.0787

________

# b)

# $P(\text{Fewer than 190 tosses}) = P(S_{190}\geq 700)$

# $E(S_{190}) = 190 \cdot \frac{7}{2} = 665$

# $\sigma_{S_{190}} = \sqrt{190\cdot \frac{35}{12}} = 23.541$

# $\implies \frac{700 - 665}{23.541} = 1.487$

# $\implies P(S_{190}\geq 700) = \int_{1.487}^{\infty}\phi(x)dx$

In [28]:
1 - norm.cdf(1.487)

0.06850740547346823

# Therefore the probability of it taking fewer than 190 rolls is 0.06851

______

# c)

### $P(\text{Between 180 and 210 tosses}) = 1 - P(\text{Fewer than 180 tosses}) - P(\text{More than 210 tosses})$

# $= 1 - 0.07866025351868966 - 0.06850740547346823 = 0.8528$